applications/callflow/src/cf_util.erl

```diff
@@ -733,6 +741,26 @@ maybe_apply_dialplan([Regex|Regexs], DialPlan, Number) ->
             <<Prefix/binary, Root/binary, Suffix/binary>>
     end.
 
+-spec load_system_dialplans(ne_binaries()) -> wh_json:object().
+load_system_dialplans(Names) ->
+    LowerNames = [wh_util:to_lower_binary(Name) || Name <- Names],
+    case whapps_config:get_all_kvs(<<"dialplans">>) of
+        Plans when is_list(Plans) ->
+            lists:foldl(
+              fun({Key, Val}, Acc) ->
```

Comment:
Move this `fun` into its own private named function, with a name starting with `fold_`